api/src/main/java/org/eclipse/microprofile/lra/annotation/Complete.java

```diff
@@ -27,44 +27,138 @@
 import java.time.temporal.ChronoUnit;
 
 /**
- * When a resource method executes in the context of an LRA and if the containing
- * class contains a method annotated with `@Compensate` then the resource will
```

Comment:
sorry, I noticed just now. We should keep the reference to `@Compensate` because otherwise without it `@Complete` won't get invoked.